api/db/seeds/data/organizations-sco-builder.js

```diff
@@ -208,7 +208,22 @@ module.exports = function organizationsScoBuilder({ databaseBuilder }) {
     externalId: '1237457C',
     provinceCode: '12',
   });
+
+  const agricultureCFA = databaseBuilder.factory.buildOrganization({
+    id: 8,
+    type: 'SCO',
+    name: 'Lycée Agricole & CFA',
```

Comment:
```suggestion
    name: 'CFA Agricole',
```
si on veut être tatillon 😛